src/mlpack/core/dists/regression_distribution.cpp

```diff
@@ -33,7 +34,7 @@ void RegressionDistribution::Train(const arma::mat& observations)
 /**
  * Estimate parameters using provided observation weights.
  *
- * @param weights probability that given observation is from distribution
+ * @param weights probability that given observation is from distribution.
```

Comment:
I'd vote for a capital letter word at the beginning of the sentence. Here and below.